### ISSUE TITLE
Support for Arm64

### ISSUE THREAD

**Issue by stanBienaives** [zakird/wkhtmltopdf_binary_gem#121]
Hello, 

Thanks for your work.
Do you have plan to support arm64 ?  
I am using wicked_pdf in a debian docker on a M1 macbook) 

The fall back for 
```
os =>  debian_9
RbConfig::CONFIG['host_cpu’]  => arch64
```
is the suffix`debian_9_i386` which throw the error: 
```Command Error: qemu-i386: Could not open '/lib/ld-linux.so.2': No such file or directory```

For those in the same situation a **workaround**  is to install the arm64 build for your distro from
https://wkhtmltopdf.org/downloads.html

install it with
```sh
apt install /path/to/debfile.deb
```

And configure wiked_pdf with 
```ruby
WickedPdf.config = {
  exe_path: '/usr/local/bin/wkhtmltopdf',
}
```

**Comment by pedrofurtado** [comment]
Hello, @stanBienaives ! 👋 

Thanks for get in contact with us.

At this moment, the gem does not provide the binary for Debian 9 Arm64, only Amd64 (and also i386).

Just for clarify: Do you have some production system hosted in Debian 9 Arm64, that uses (or want to use) wkhtmltopdf gem? 👍

**Comment by pedrofurtado** [comment]
I checked in Wkhtmltopdf official site, there is a binary available for Debian 9 arm64. I will try to spend some time for this support 🎉

**Comment by pedrofurtado** [comment]
I check that Wkhtmltopdf official site provides binary for debian 9 arm64. I will try spend some time for this support 🎉

**Comment by jbosse** [comment]
12.6 also has support for M1 Macs https://github.com/wkhtmltopdf/wkhtmltopdf/releases/tag/0.12.6

I think this would also require the addition of `macos_carbon` to https://github.com/zakird/wkhtmltopdf_binary_gem/blob/5645e81913858865b416ea19afaab3dd43ee663f/bin/wkhtmltopdf#L50

**Comment by unixmonkey** [comment]
Many arm64 binaries are now included in version `0.12.6.7`. Please check it out and let me know if what you need is still missing.
